docs/content/latest/integrations/spring-framework/sdyb.md

```diff
@@ -0,0 +1,141 @@
+---
+title: Spring Data YugabyteDB
+linkTitle: Spring Data YugabyteDB
+description: Spring Data YugabyteDB
+aliases:
+menu:
+  latest:
+    identifier: sdyb
+    parent: spring-framework
+    weight: 578
+isTocNested: true
+showAsideToc: true
+---
+
+Spring Data modules are used for accessing databases and performing various tasks via Java APIs, therefore eliminating the need to learn a database-specific query language.
+
+Spring Data YugabyteDB (SDYB) modules provide support for YSQL APIs and enable you to build cloud-native applications.
+
+## Overview
+
+The following are some of the features included in SDYB YSQL:
+
+- Spring Data `YsqlTemplate` and `YsqlRepository` for querying YugabyteDB
+- `@EnableYsqlRepostiory` annotation for enabling `YsqlRepository`
+- Yugabyte Distributed SQL transaction manager
+- Spring Boot starter support for YugabyteDB
+- Cluster awareness achieved via elimination of load balancer from SQL
+- Topology awareness necessary for developing geographically-distributed applications
+- Partition awareness for achieving row-level geo-partitioning
+
+For more information, demonstrations, and contribution guidelines, see [Spring Data YugabyteDB GitHub project.](https://github.com/yugabyte/spring-data-yugabytedb/) 
+
+## Project Dependencies
+
+The project definition includes the following dependencies:
+
+```properties
+<dependency>
+    <groupId>com.yugabyte</groupId>
+    <artifactId>spring-data-yugabytedb-ysql</artifactId>
+    <version>2.3.0</version>
```

Comment:
Please update the version to the following, I'll reach out to update the version after GA release.

```
<version>2.3.0-RC1.b5</version>
```

Reply:
Changed